collections/_posts/2025-05-06-wcag-21.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "WCAG 2.1 published with minor technical fixes"
+date: 2025-05-06
+---
+
+W3C published an update that addresses minor technical issues in the December 2024 publication of WCAG 2.1. The 6 May 2025 publication is [Web Content Accessibility Guidelines (WCAG) 2.1.](https://www.w3.org/TR/2025/REC-WCAG21-20250506/)
```

Comment:
```suggestion
W3C published an update that addresses minor technical issues in the December 2024 publication of WCAG 2.1. The 6 May 2025 publication is [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/2025/REC-WCAG21-20250506/).
```
@shawna-slh @tamsinewing555 I think the period should not be in the link text.

Reply:
+1 !